grammars/prql-lezer/src/prql.grammar

```diff
@@ -96,7 +97,7 @@ Op_bin { Op_bin_only | Op_unary }
 
 Table_def { @specialize<ident_part, "let"> ident_part "=" Nested_pipeline ( newline+ | end ) }
 
-Func_def { @specialize<ident_part, "func"> Func_def_name Func_def_param* "->" expr_call ( newline+ | end ) }
+Func_def { @specialize<ident_part, "let"> Func_def_name "=" Func_def_param* "->" expr_call ( newline+ | end ) }
```

Comment:
Does this handle the ambiguity with Table_def OK? https://github.com/PRQL/prql/pull/2868/files#diff-b9ff79077300e8e401d4b8b3fe1f55d89a0ea58ad1582012ef7f6ce867667f57R98 

I wasn't sure if you've tried it in the playground. If so, then great. If not, we could them consolidate to `Def`, and offer an option of `lambda` as a RHS, which would be `Func_def_param* "->" expr_call ( newline+ | end )`...

Reply:
This is a far more difficult task than I thought it would be, and it is out of my hands.

Added comment as TODO.
Currently this does not support function definitions and relation literals.
Unfortunately I could not find a solution to this after reading your comment.

You can merge this not working or close this and wait for someone else to work on this.

Reply:
No problem! I just Approved before seeing your comment, I agree it's better to move forward!